prereise/gather/demanddata/transportation_electrification/data_process.py

```diff
@@ -0,0 +1,132 @@
+import pandas as pd
+
+#Download CSV of NHTS 2017 from https://nhts.ornl.gov/
+
+def data_filtering(census_division, filepath: str = "trippub.csv"):
+    """Filter raw NHTS data to be used in mileage.py
+    
+    :param str filepath: filepath for the "trippub.csv" file in downloaded NHTS data
+    :return: (*pd.DataFrame*) -- filtered and sorted trip data
+    """
+
+    nhts_raw = pd.read_csv(filepath)
```

Comment:
@BainanXia and @danielolsen, I believe we want this file on our blob storage, read from there and refer to it in the **ATTRIBUTION** file. I believe, we could have a year parameter that would get the file for a given year and raise an error if it is not available on the blob storage.